### PR TITLE
Created stub file for icons adjustments. Added ramp sizes.

### PR DESCRIPTION
**Pull request by wdo3650**
Created stub file for icons adjustments. Added ramp sizes to icons sass file.

### CODE REVIEW

File: src/sass/_Fabric.Icons.scss [hunk 1]
```diff
@@ -400,4 +400,21 @@
...
\ No newline at end of file
+}
+
+// Fabric Core Icon variables
+$ms-Icon-12: 12px;
+$ms-Icon-16: 16px;
+$ms-Icon-20: 20px;
+
+.ms-Icon-12 {
+  font-size: 12px;
```
Comment (mikewheaton) [inline src/sass/_Fabric.Icons.scss]: I think you meant to use the variables from above here, right? Also, these should be modifier classes like `ms-Icon--12` to be used in conjunction with `ms-Icon`.

**Comment by mikewheaton** [comment]
Approved 🚕

**Comment by wdo3650** [comment]
@mikewheaton getting this same error when I nuke my node modules and npm install.

**Comment by mikewheaton** [comment]
Once PR #676 gets merged into master, you can merge master into this branch to resolve the build issues.

**Comment by wdo3650** [comment]
I just noticed this would be a merge into master as opposed to the beta/mdl2 branch. I'm going to close this and reopen for beta/mdl2

**Pull request closed without merging**